lib/helper/Puppeteer.js

```diff
@@ -1985,6 +1985,29 @@ class Puppeteer extends Helper {
   async grabDataFromPerformanceTiming() {
     return perfTiming;
   }
+
+  /**
+   * {{> grabElementSize }}
+   *
+   */
+  async grabElementBoundingRect(locator, prop) {
+    let els = await this._locate(locator);
+    els = await Promise.all(els.map(el => el.boundingBox()));
+    const result = [];
+    els = els.filter(v => v);
```

Comment:
what this line do?

Reply:
this line to remove empty items, null, etc.